kotlin-source/build.gradle

```diff
@@ -32,15 +32,17 @@ dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.11'
 
```

Comment:
And these dependencies should probably be excluded from both Java and Kotlin:
```
configurations.compile {
    // We don't need these because we already include netty-all.
    exclude group: 'io.netty', module: 'netty-transport'
    exclude group: 'io.netty', module: 'netty-handler'

    // We are using SLF4J's implementation instead.
    exclude group: 'commons-logging', module: 'commons-logging'
}
```